lua/drop/config.lua

```diff
@@ -121,13 +490,123 @@ M.themes = {
       "🥥",
       "🌊",
     },
-    colors = {
-      "#236e96",
-      "#15b2d3",
-      "#ffd700",
-      "#f3872f",
-      "#ff598f",
+    colors = { "#236e96", "#15b2d3", "#ffd700", "#f3872f", "#ff598f" },
+  },
+  temporal = {
+    symbols = { "🕐", "🕑", "🕒", "🕓", "🕔", "🕕", "🕖", "🕗", "🕘", "🕙", "🕚", "🕛" },
+    colors = { "#8B0000", "#B22222", "#DC143C", "#FF4500", "#FF6347" },
+  },
+  us_thanksgiving = {
+    symbols = {
+      "🦃",
+      "🍂",
+      "🍁",
+      "🌽",
+      "🥧",
+      "🍠",
+      "🍎",
+      "🍖",
+      "🍗",
+      "🥖",
+      "🥔",
+      "🍇",
+      "🍷",
+      "🌰",
+      "🥕",
     },
+    colors = { "#8B4513", "#D2691E", "#FF4500", "#228B22", "#A52A2A" },
+  },
+  travel = {
+    symbols = { "✈️", "🌍", "🗺️", "🏨", "🧳", "🗽", "🚂" },
+    colors = { "#FFD700", "#87CEEB", "#FF4500", "#2E8B57", "#20B2AA" },
+  },
+  tropical = {
+    symbols = { "🌴", "🍍", "🍉", "🥥", "🌺", "🐢", "🌊" },
+    colors = { "#20B2AA", "#FF6347", "#FFD700", "#FF4500", "#8A2BE2" },
+  },
+  urban = {
+    symbols = { "🏢", "🚕", "🚇", "🍕", "🚦", "🛴", "🎧" },
+    colors = { "#A9A9A9", "#2F4F4F", "#B22222", "#FF8C00", "#D3D3D3" },
+  },
+  us_independence_day = {
+    symbols = {
+      "🇺🇸",
+      "🎆",
+      "🗽",
+      "🦅",
+      "🌭",
+      "🍔",
+      "⭐",
+      "🎉",
+      "🥳",
+      "🍻",
+      "🥁",
+      "🎵",
+      "🎶",
+      "🚀",
+      "💥",
+    },
+    colors = { "#B22234", "#3C3B6E", "#FFFFFF", "#FFD700", "#FF6347" },
+  },
+  valentines_day = {
+    symbols = {
+      "❤️",
+      "💖",
+      "💘",
+      "💝",
+      "💕",
+      "💓",
+      "💞",
+      "💟",
+      "💌",
+      "🌹",
+      "🍫",
+      "💐",
+      "💍",
+      "🍷",
+      "🕯️",
+    },
+    colors = { "#FF6B8D", "#FFB6C1", "#DD2A7B", "#FFD1DC", "#8B0000" },
+  },
+  wilderness = {
+    symbols = { "🌲", "🐺", "🦌", "🏞️", "🔥", "⛺", "🌌" },
+    colors = { "#228B22", "#A52A2A", "#DEB887", "#696969", "#B0E0E6" },
+  },
+  wildwest = {
+    symbols = { "🤠", "🐎", "🌵", "🔫", "⛏️", "🌄", "🚂" },
+    colors = { "#8B4513", "#BDB76B", "#FFD700", "#D2691E", "#FFDEAD" },
+  },
+  winter_wonderland = {
+    symbols = { "❄️", "⛄", "🌨️", "🎿", "🛷", "🏔️", "🧣" },
+    colors = { "#ADD8E6", "#B0E0E6", "#87CEFA", "#1E90FF", "#6495ED" },
+  },
+  xmas = {
+    symbols = {
+      "🎄",
+      "🎁",
+      "🤶",
+      "🎅",
+      "🛷",
+      "❄",
+      "⛄",
+      "🌟",
+      "🦌",
+      "🎶",
+      "❄️ ",
+      " ",
+      "❅",
+      "❇",
+      "*",
+    },
+    colors = { "#146B3A", "#F8B229", "#EA4630", "#BB2528" },
+  },
+  zodiac = {
+    symbols = { "♈", "♉", "♊", "♋", "♌", "♍", "♎", "♏", "♐", "♑", "♒", "♓" },
+    colors = { "#FF4500", "#FF6347", "#FFA07A", "#FFD700", "#FFFAF0" },
+  },
+  zoo = {
+    symbols = { "🦁", "🐘", "🦓", "🦒", "🦅", "🦉", "🐆" },
+    colors = { "#A0522D", "#8B4513", "#D2B48C", "#BDB76B", "#D2691E" },
```

Comment:
I added many new themes and also ordered them alphabetically by theme name to make them easier to navigate.